en/manage-cli-comp-hw.md

```diff
@@ -156,6 +154,7 @@ Where RESOURCES is a space-separated list from:
 **cpu_speed=**requested minimum cpu speed in MHz  
 **memory=**requested memory in MB  
 **architecture=**requested architecture that pod must support  
+**storage=**requested stroage
```

Comment:
The user can now also use an `interfaces` constraint here (with the same syntax as is used when allocating a machine) in order to allocate an IP address or select a space, subnet, or VLAN.